src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -747,7 +746,7 @@ public void BuildApplicationAndClean ([Values (false, true)] bool isRelease, [Va
 			proj.SetProperty ("AndroidPackageFormat", packageFormat);
 			if (packageFormat == "aab")
 				// Disable the shared runtime for aabs because it is not currently compatible and so gives an XA0119 build error.
```

Comment:
You might update the above comment:
```suggestion
				// Disable fast dev for aabs because it is not currently compatible and so gives an XA0119 build error.
```